src/dune_rules/gen_rules.ml

```diff
@@ -89,7 +89,11 @@ end = struct
       in
       empty_none
     | Executables exes when Expander.eval_blang expander exes.enabled_if ->
-      Option.iter exes.install_conf ~f:files_to_install;
+      let* () =
```

Comment:
There is `Memo.Option.iter` if you'd like to make this shorter. (Although I'm personally still undecided if `Memo.Option` and `Memo.Result.iter` are really worth special support.)

Reply:
Since we have this function, let's do it to keep the style consistent

Reply:
updated